airflow/www/static/js/trigger.js

```diff
@@ -20,13 +20,24 @@
 /* global document, CodeMirror, window */
 
 const textArea = document.getElementById('json');
+const recentConfigList = document.getElementById('recent_configs');
 const minHeight = 300;
 const maxHeight = window.innerHeight - 450;
 const height = maxHeight > minHeight ? maxHeight : minHeight;
 
 CodeMirror.fromTextArea(textArea, {
   lineNumbers: true,
-  mode: { name: 'javascript', json: true },
+  mode: {
+    name: 'javascript',
+    json: true,
+  },
   gutters: ['CodeMirror-lint-markers'],
   lint: true,
-}).setSize(null, height);
+})
+  .setSize(null, height);
+
+function setRecentConfig(e) {
+  document.querySelector('.CodeMirror').CodeMirror.setValue(e.target.value.replaceAll("'", '"'));
```

Comment:
To avoid manually replacing the quotes, you can `json.dumps(conf)`. See what is done in the grid_data endpoint, and my other comments.